packages/protocol/README.md

```diff
@@ -26,43 +26,45 @@ If a new contract needs to be deployed, create a migration file in the `migratio
 To apply any new migrations to a network, run:
 
 ```bash
-yarn run migrate -n NETWORK
+yarn run devchain:migrate -n NETWORK
 ```
 
 ### Accounts
 
 To give an account some gold, wrapped gold, and stable token, run:
 
+[//]: <> (FIXME: faucet does not exist)
```

Comment:
Outdated -> yarn script is missing

Reply:
I don't think we support this anymore rather we either have https://faucet.celo.org/alfajores or it is expected to for users to reach out to cLabs and request for larger amounts of Celo - can be deleted